EPMLSTRTAI-337 change urlsValidation.

iteaky opened the pull request:
Now it's collect all Exception and then throw them in one.

iteaky committed:
EPMLSTRTAI-337 change urlsValidation. Now it's collect all Exception and the throw them in one.

pomkine reviewed (changes requested):
Would be perfect to have a couple of tests here ;)

iteaky committed:
EPMLSTRTAI-337 Test for urls validator that our exception message contains our invalid urls

iteaky committed:
EPMLSTRTAI-337 add validation on not contains valid url in exception

iteaky committed:
EPMLSTRTAI-337 codacy fixes
1)Fields ""@Rule" should be declared at the top of the class, before any method declarations, constructors,initializers or inner classes.

2)Method names should not contain underscores
